assets/js/modules/analytics/components/settings/SettingsView.js

```diff
@@ -37,6 +38,7 @@ const { useSelect } = Data;
 export default function SettingsView() {
 	const accountID = useSelect( ( select ) => select( STORE_NAME ).getAccountID() );
 	const propertyID = useSelect( ( select ) => select( STORE_NAME ).getPropertyID() );
+	const propertyID4 = useSelect( ( select ) => select( MODULES_ANALYTICS_4 ).getPropertyID() );
```

Comment:
We should rely on `const isGA4Enabled = useFeature( 'ga4setup' );` when we interact with the `analytics-4` module functionality. In other words we should try to pull `propertyID` only if the `isGA4Enabled` is true.